CHANGELOG.md

```diff
@@ -0,0 +1,12 @@
+## next major release
+
+* Makes commands `changelog` and `upload` compatible with Shopware 6 plugins.
+* Makes the command `compatibility` fail when executed for a Shopware 6 plugin.
+* Fixes a bug that lead to a crash when trying to dump a plugin that does not exist.
```

Comment:
What exactly do you mean by 'dump' here? Was this crash in a specific command?

Reply:
Command `dump-plugin`.